keyboards/keychron/k8/rgb/rules.mk

```diff
@@ -0,0 +1,6 @@
+# Build Options
+#   comment out to disable the options.
+#
+
```

Comment:
this is getting picked up by the linter and failing
```☒ Lint check failed for: keychron/k8/rgb```
remove the rules.mk from here and it will shut up

Reply:
bugger.. of course.. Why didn't I think of that :laughing: